tasks.py

```diff
@@ -254,6 +254,7 @@ def checkemulation(c):
         "adf5610",
         "adg2128",
         "adis16460",
+        "adis16475",
```

Comment:
We don't allow new parts to be added without emulation support , unless for very specific reasons. Can you provide a context file for the system (see here https://analogdevicesinc.github.io/pyadi-iio/dev/index.html#new-hardware-requirements and https://analogdevicesinc.github.io/pyadi-iio/dev/index.html#emulation)?

Reply:
Done

Reply:
Please remove from this list.